tests/unit/views/services/camera_test.js

```diff
@@ -20,6 +19,11 @@ describe('Camera service view tests', function() {
     component = TestUtils.renderIntoDocument(
       <CameraServiceView service={serviceMock} foxbox={foxboxStub}/>
     );
+
+    assert.isDefined(
+      component,
+      'ReactDOM.render() did not render component (see issue #133).'
```

Comment:
Can you use the link, an issue number only can be ambiguous (is it a foxbox error? a React error?):
`https://github.com/fxbox/app/issues/133`